.github/workflows/docs-deploy-test.yml

```diff
@@ -0,0 +1,24 @@
+name: Test Docs deployment
+
+on:
+  pull_request:
+    branches:
+      - feature/docusaurus-based-docs
```

Comment:
We will need to update this before merging

Reply:
Absolutely, but I needed that to test the gh-actions integration properly